### PR TITLE
QUEST #1158-1171 rev1

### PR DESCRIPTION
**Pull request by vtange**
Modestas Story continuation. Will do 1171 onwards in another PR or something to avoid confusion

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
...
+QUEST_20150317_001158	Anyhow, as Modestas and the Old Lady parted ways, he knew he now had a dangerous power guaranteeing his victory at any time in the near future. {nl} He walked, looking for the next person to bet huge on. As he wondered how much to bet on his next move, he heard the sound of running footsteps behind him.
+QUEST_20150317_001159	Modestas looked back, it was his his good old friend Sigfried on horseback riding towards him. {nl} Sigfried and Modestas knew each other, as both had been together through the highs and lows of gambling. They greeted each other, and talked about where they were going. To walk with his good friend Modestas, Sigfried got off his horse. They walked at a leisurely pace, with idle chatter along the way. {nl}
```
Comment (grillo-delmal) [inline QUEST.tsv]: ...it was **his his** good old friend ...

File: QUEST.tsv [hunk 2]
```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
...
+QUEST_20150317_001160	However, before long, Modestas could no longer continue walking with Sigfried. He was fully aware of his deadline, and had no time for trivial activities with Sigfried. {nl}
+QUEST_20150317_001161	Even during their normal everyday talk, Modestas and Sigfried unconsciously made big and small bets. Eventually, one would utter the word, "bet?" without knowing he did. {nl} Both Sigfried and Modestas weren't rich, therefore, there was no way this bet could be big. Knowing this could trigger the curse, Modestas really wanted to simply tell Sigfried he's gotta go. {nl} In the end, Modestas uttered these words. {nl}
+QUEST_20150317_001162	"Are you not busy?" You seemed busy on your horse before you passed by for a talk. You should stop worrying about me and go." {nl} Sigfried, hearing Modestas's words, replied, {nl} "I was busy, but I still have time to walk with you for another 10 minutes or so. This horse might look look average, but when it comes to it he can go pretty fast." {nl}
```
Comment (ghost) [inline QUEST.tsv]: "Are you not busy?" -> "Are you not busy?
There's an extra quotation mark there.

look look -> look

File: QUEST.tsv [hunk 3]
```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
...
+QUEST_20150317_001163	Seeing Sigfried not budge, Modestas took another shot at persuading him to go, {nl} Wouldn't it be better to be early as opposed to be on time?" {nl} {nl} Hearing this, Sigfried became adamant, {nl} "Do you not believe what I just said? Or are you wilfully ignoring it all?" {nl} A disconcerted Modestas said, {nl} "No, I didn't mean it like that……" {nl}
```
Comment (ghost) [inline QUEST.tsv:1163]: Wouldn't it be better -> "Wouldn't it be better

Missing quotation mark.

File: QUEST.tsv [hunk 4]
```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
...
+QUEST_20150317_001164	However, before Modestas could say any more, Sigfried said, "If you don't believe me that much, why not have a bet?" {nl} {nl} Naturally, the words, "Sure. what do you want to bet?" rose up in Modestas' throat.
```
Comment (ghost) [inline QUEST.tsv:1164]: "Sure. what do you want to bet?" rose up in Modestas' throat. -> "Sure. What do you want to bet?" rose up in Modestas' throat.

Capitalize "what"

File: QUEST.tsv [hunk 5]
```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
...
+QUEST_20150317_001167	As soon as the bet started, Sigfried tried every method possible to command the horse to move. Despite his efforts it didn't move a single hoof at all.{nl}At the end, Sigfried repeatedly cursed its strange behavior. Realizing he will be late to his appointment without a horse, he threw the reins to Modestas and sprinted away.{nl} As Sigfried disappeared, Modestas felt uncomfortable.
+QUEST_20150317_001168	For Modestas, who had always been elated upon winning bets and depressed upon losing them, to win and feel depressed was a first in his entire life. {nl}
+QUEST_20150317_001169	Knowing he had just deceived his long-time gambling friend Sigfried, and also missed an opportunity to earn a lot of money, Modestas felted conflicted, and walked slowly alongside the horse he had just won. {nl} While walking, Modestas remembered the earlier warnings of his unconditional bet results. He realized that now with his upcoming defeat, he is doomed to not earn any more silver than he started with.
```
Comment (ghost) [inline QUEST.tsv]: felted -> felt

File: QUEST.tsv [hunk 6]
```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
...
-QUEST_20150317_001170	People who model Stars Mr. encounters that were walking down the street and pull to say so {nl} was the soldiers of the people number. Soldiers to {nl} model Stars san see looked to the hassle to put something. {nl} model Stars Mr., was Jinagaryo only whether they are not, soldiers cried when you see the model's Stars. {nl}
-QUEST_20150317_001171	There will pull the words to Uncle. Come, do not me help us. {nl} model Stars Mr., the number of the soldiers did not know what helped me in any thing, the sound to hear it went closer to them to collect the words, rather than be able to pass as it is. {nl} Once the model Stars Mr. approaches, one of the soldiers said. {nl} "just good come small. Cattle Once you have me look at the referee that we were one of the bet here." {nl}
+QUEST_20150317_001158	Anyhow, as Modestas and the Old Lady parted ways, he knew he now had a dangerous power guaranteeing his victory at any time in the near future. {nl} He walked, looking for the next person to bet huge on. As he wondered how much to bet on his next move, he heard the sound of running footsteps behind him.
```
Comment (Soukyuu) [inline QUEST.tsv]: It says "footsteps" here, but the next line says Sigfried is riding on a horse. Shouldn't it be something along the lines of "sound of hooves" instead?

File: QUEST.tsv [hunk 7]
```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
...
+QUEST_20150317_001159	Modestas looked back, it was his good old friend Sigfried on horseback riding towards him. {nl} Sigfried and Modestas knew each other, as both had been together through the highs and lows of gambling. They greeted each other, and talked about where they were going. To walk with his good friend Modestas, Sigfried got off his horse. They walked at a leisurely pace, with idle chatter along the way. {nl}
+QUEST_20150317_001160	However, before long, Modestas could no longer continue walking with Sigfried. He was fully aware of his deadline, and had no time for trivial activities with Sigfried. {nl}
+QUEST_20150317_001161	Even during their normal everyday talk, Modestas and Sigfried unconsciously made big and small bets. Eventually, one would utter the word, "bet?" without knowing he did. {nl} Both Sigfried and Modestas weren't rich, therefore, there was no way this bet could be big. Knowing this could trigger the curse, Modestas really wanted to simply tell Sigfried he's gotta go. {nl} In the end, Modestas uttered these words. {nl}
```
Comment (Soukyuu) [inline QUEST.tsv]: "Gotta go" is very colloquial. Shouldn't we use a more formal wording?

File: QUEST.tsv [hunk 8]
```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
...
+QUEST_20150317_001163	Seeing Sigfried not budge, Modestas took another shot at persuading him to go, {nl} Wouldn't it be better to be early as opposed to be on time?" {nl} {nl} Hearing this, Sigfried became adamant, {nl} "Do you not believe what I just said? Or are you wilfully ignoring it all?" {nl} A disconcerted Modestas said, {nl} "No, I didn't mean it like that……" {nl}
+QUEST_20150317_001164	However, before Modestas could say any more, Sigfried said, "If you don't believe me that much, why not have a bet?" {nl} {nl} Naturally, the words, "Sure. what do you want to bet?" rose up in Modestas' throat.
+QUEST_20150317_001165	Considering Modestas' habit, to simply not immediately accept a gamble took superhuman willpower from him. {nl} "What…  what kind of bet?" {nl} "We were going to bet on whether my horse is fast or not." {nl} Modestas desperately fought in his inner gambler.
```
Comment (Soukyuu) [inline QUEST.tsv:1165]: "fought in his inner gambler" -> "fought his gambler nature"?
I'm not familiar with such a use of "in" in this context.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**